packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -0,0 +1,339 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BalanceTransferPayload, TransferDomainType } from '../../../src/category/account'
+import { RpcApiError } from '@defichain/jellyfish-api-core/dist/index'
+import BigNumber from 'bignumber.js'
+
+describe('TransferDomain', () => {
+  let dfiAddress: string, ethAddress: string
+  const amountToTransfer = 5
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+    await client.masternode.setGov({
+      ATTRIBUTES: {
+        'v0/params/feature/evm': 'true'
+      }
+    })
+    await container.generate(1)
+    dfiAddress = await container.call('getnewaddress')
+    // transfer utxostoaccount
+    await container.call('utxostoaccount', [{ [dfiAddress]: '100@0' }])
+    await createToken(dfiAddress, 'BTC', 200)
+    await container.generate(1)
+    await container.generate(1)
+    ethAddress = await container.getNewAddress('', 'eth')
```

Comment:
Do we have tests for this? Maybe not on this PR but on another PR. I think we should have a test that the generated eth address is a valid eth address.

Reply:
Added test case for eth address generation